enterprise/server/configure.md

```diff
@@ -334,13 +367,20 @@ ephox {
 }
 ```
 
+## Spelling service settings
+
 ### `spelling` (optional)
 
-The Spelling service has two configurable settings:
+The Spelling service has three configurable settings:
```

Comment:
You'll save yourself future time by just changing this to a "these" , or "a number of"

Reply:
```suggestion
The Spelling service has these configurable settings:
```